packages/connect/e2e/tests/device/passphrase.test.ts

```diff
@@ -21,12 +21,14 @@ describe('TrezorConnect passphrase', () => {
     beforeAll(async () => {
         await setup(controller, {
             mnemonic: 'mnemonic_all',
-            passphrase_protection: true,
+            use_passphrase: true, // note: this will not work with 2-master and user-env
```

Comment:
ok, we can live with it for now. later it will be fixed either in trezor-user-env or handled globally in packages/trezor-user-env-link

Reply:
yup, just revert this change later